terrain_simplification_ros/package.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>terrain_simplification_ros</name>
+  <version>1.0.0</version>
+  <description>
+    A ROS-based interface to terrain_simplification package.
+  </description>
+
+  <author>Oliwier Melon</author>
+  <maintainer email="omelon@robots.ox.ac.uk">Oliwier Melon</maintainer>
+  <license>BSD</license>
+  
+  <buildtool_depend>catkin</buildtool_depend>
+
+  <depend>message_runtime</depend>
+  <depend>message_generation</depend>
+  <depend>std_msgs</depend>
+  <depend>geometry_msgs</depend>
+  <depend>roscpp</depend>
+  <depend>rosbag</depend>
+  <depend>grid_map_core</depend>
+  <depend>grid_map_cv</depend>
+  <depend>grid_map_filters</depend>
+  <depend>grid_map_filters_drs</depend>
```

Comment:
Note, this is a private repository. I understand this PR has been stale for a while and we need to clean it up at some point - but adding a private dependency for a public repository won't work. One way is to move the runtime config + launch file to the user repository and not use it by default in the config included here

Reply:
I agree; I admit, I have not thought about it. 

If we do what you suggest, can a filter from `grid_map_filters_drs` still be loaded by `filter_chain` even if the repository is not added as a dependency? If so, I am happy to include the specific `config.yaml` and `.launch` files in e.g. RHECALL.

Reply:
To my knowledge, yes (the dependencies need to be at the level of the package that has the launch file / config), the point of using `pluginlib` is to be able to split packages and load plugins (here filters) dynamically

Reply:
Okay, I will proceed as you suggested.

Something to bear in mind is that we will need to revisit the same problem when we make RHECALL publicly available.

Reply:
Let's cross that bridge when we get to it :-)